src/operations/create_collection.ts

```diff
@@ -41,6 +41,17 @@ export interface TimeSeriesCollectionOptions extends Document {
   granularity?: 'seconds' | 'minutes' | 'hours' | string;
 }
 
+/** @public
+ * Configuration options for clustered collections
+ * @see https://www.mongodb.com/docs/v5.3/core/clustered-collections/
+ */
+export interface ClusteredCollectionOptions extends Document {
+  name?: string;
+  key?: Document;
+  unique?: boolean;
```

Comment:
Right now unique is required and has to be `unique: true`, but it seems like there are plans for this in the future. If you leave it out or set it to anything else the server will complain.